internal/pkg/term/selector/selector.go

```diff
@@ -8,15 +8,22 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/aws/copilot-cli/internal/pkg/aws/ec2"
 	"github.com/aws/copilot-cli/internal/pkg/config"
 	"github.com/aws/copilot-cli/internal/pkg/term/color"
 	"github.com/aws/copilot-cli/internal/pkg/term/log"
 	"github.com/aws/copilot-cli/internal/pkg/term/prompt"
 )
 
+var (
+	ErrVpcNotFound     = fmt.Errorf("no VPC found, create a new VPC first to import\n or run %s to use default Copilot environment config please", color.HighlightCode("env init"))
```

Comment:
```suggestion
	ErrVPCNotFound     = errors.New("no existing VPCs found")
```
Instead of having the recommended action to be part of the err message, can we decouple them?
```go
if err := s.VPC(prompt, sel); err != nil {
   if err == selector.ErrVPCNotFound {
       log.Errorf(`No existing VPCs were found. You can either:
- Create a new VPC first and then import it.
- Use the default Copilot environment configuration.
`)
   }
   return err
}
```